src/python/bot/tasks/corpus_pruning_task.py

```diff
@@ -218,10 +218,11 @@ def sync_to_disk(self):
       raise CorpusPruningException('Failed to sync corpus to disk.')
 
     if not self.quarantine_corpus.rsync_to_disk(self.quarantine_corpus_path):
-      raise CorpusPruningException('Failed to sync quarantine corpus to disk.')
+      logs.log_error('Failed to sync quarantine corpus to disk.')
```

Comment:
can you add ", fuzz_target=self.fuzz_target" to both places, makes it easier to debug :)

Reply:
Hm, I feel like last time I did that, @oliverchang told me to remove, as it's auto added. Am I mistaken?

Reply:
This is not needed in launchers since we add it in logs.configure,

see
  logs.configure(
      'run_fuzzer', {
          'fuzzer': fuzzer_name,
          'engine': 'libFuzzer',
          'job_name': environment.get_value('JOB_NAME')
      })

but this is corpus pruning task, so it wont have this fuzzer var set.

Reply:
Ah, I see, TIL! Thank you!